eta/core/vgg16.py

```diff
@@ -434,33 +425,27 @@ def _build_fc_layers(self):
             self.fc3l = tf.nn.bias_add(tf.matmul(self.fc2, fc3w), fc3b)
             self.parameters += [fc3w, fc3b]
 
-    def _load_weights(self, weights_config, sess):
-        weights = Weights(weights_config)
+    def _load_model(self, model, sess):
+        weights = etam.NpzModelWeights(model).load()
         for i, k in enumerate(sorted(weights)):
-            logger.debug("%s %s %s", i, k, np.shape(weights[k]))
             sess.run(self.parameters[i].assign(weights[k]))
 
 
-class VGG16FeaturizerConfig(Config):
+class VGG16FeaturizerConfig(VGG16Config):
     '''Configuration settings for a VGG16Featurizer that works on images.'''
-
-    def __init__(self, d):
-        self.weights = self.parse_object(
-            d, "weights", WeightsConfig, default=None)
-        if self.weights is None:
-            self.default_config = VGG16Config.load_default()
-            self.weights = self.default_config.weights
+    pass
```

Comment:
I suppose it is fine to keep this, but seems pointless now.  The earlier implementation let the user specify a different weights for a featurizer, but I don't think that functionality was ever used.

Reply:
It's still possible to specify different weights